tests/bugfixes/github/test_issue_1099.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+
+from system_tests import CaseMeta, path, check_no_ASAN_UBSAN_errors
+
+
+class EmptyValueInCommandFile(metaclass=CaseMeta):
+    """
+    Regression test for the bug described in:
+    https://github.com/Exiv2/exiv2/issues/1099
+
+    An empty value in the command file causes a std::out_of_range exception.
+    """
+    url = "https://github.com/Exiv2/exiv2/issues/1099"
+
+    filename1 = path("$data_path/issue_1099_poc.txt")
+    filename2 = path("$data_path/issue_1099_poc.bin")
+    commands = ["$exiv2 -m $filename1 mo $filename2"]
+    stdout = ["""Usage: $exiv2exe [ options ] [ action ] file ...
```

Comment:
I would suggest to instead override `compare_stdout` as follows:
``` Python
def compare_stdout(self, i, command, got_stdout, expected_stdout):
    pass
```
That will cancel the stdout comparison, which imho is not worth checking here as it is completely unrelated.

Unless of course we want to check the stdout here too?